db/migrate/20190925070652_remove_user_to_solidus_gdpr_request.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class RemoveUserToSolidusGdprRequest < ActiveRecord::Migration[5.1]
```

Comment:
Let's leave this, but for future reference, this right phrasing is `RemoveUserFromSolidusGdprRequest` ("from" instead of "to").